### DISCUSSION TITLE
Unable to install on Mac

### DISCUSSION THREAD

**Discussion by rmclachl** [firedrakeproject/firedrake#2670] [Firedrake support]
Have upgraded to OS 12.6.1.

[firedrake-install.log](https://github.com/firedrakeproject/firedrake/files/10159458/firedrake-install.log)
[configure.log](https://github.com/firedrakeproject/firedrake/files/10159480/configure.log)

The final interaction from the installation script is:

Removing existing h5py installations

Installing h5py/
Installing libsupermesh
Cloning libsupermesh

The authenticity of host 'bitbucket.org (2406:da00:ff00::6b17:d1f5)' can't be established.
RSA key fingerprint is SHA256:zzXQOXSRBEiUtuE8AikJYKwbHaxvSc0ojez9YXaGp1A.
This key is not known by any other names
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Failed to clone libsupermesh using ssh, falling back to https.
Successfully cloned repository libsupermesh.
Checking out branch master
Successfully checked out branch master
Updating submodules.
Successfully updated submodules.
Traceback (most recent call last):
  File "/Users/rmclachl/Documents/firedrake-install", line 1868, in <module>
    build_and_install_libsupermesh()
  File "/Users/rmclachl/Documents/firedrake-install", line 1279, in build_and_install_libsupermesh
    check_call(cmd)
  File "/Users/rmclachl/Documents/firedrake-install", line 653, in check_call
    log.debug(subprocess.check_output(arguments, stderr=subprocess.STDOUT, env=os.environ).decode())
  File "/usr/local/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 421, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/local/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['cmake', '..', '-DBUILD_SHARED_LIBS=ON', '-DCMAKE_INSTALL_PREFIX=/Users/rmclachl/Documents/firedrake', '-DMPI_C_COMPILER=/Users/rmclachl/Documents/firedrake/src/petsc/default/bin/mpicc', '-DMPI_CXX_COMPILER=/Users/rmclachl/Documents/firedrake/src/petsc/default/bin/mpicxx', '-DMPI_Fortran_COMPILER=/Users/rmclachl/Documents/firedrake/src/petsc/default/bin/mpif90', '-DCMAKE_Fortran_COMPILER=/Users/rmclachl/Documents/firedrake/src/petsc/default/bin/mpif90', '-DLIBSUPERMESH_AUTO_COMPILER_FLAGS=OFF', '-DCMAKE_Fortran_FLAGS=-fallow-argument-mismatch']' returned non-zero exit status 1.

Install log saved in /Users/rmclachl/Documents/firedrake-install.log

**Comment by ksagiyam** [comment]
In the `configure.log`, we see `ld: library not found for -lSystem`. This is a typical error that we see after macOS upgrade. We suggest that you reinstall Command Line Tools.

https://github.com/firedrakeproject/firedrake/discussions/2605
